.github/CODEOWNERS

```diff
@@ -106,5 +106,16 @@ clang/test/AST/Interp/ @tbaederr
 # MLIR Sparsifier.
 /mlir/**/*SparseTensor*/ @aartbik @PeimingLiu @yinying-lisa-li @matthias-springer
 
+# MLIR NVGPU Dialect
+/mlir/**/NVGPU/ @grypp
+/mlir/**/NVGPUTo*/ @grypp
+/mlir/**/NVGPUTo*/ @grypp
+/mlir/**/sm90*/ @grypp
```

Comment:
Is this catching the integration tests? don't you want to match something like `/mlir/test/**/CUDA*/` ?

Reply:
I could catch actually all of the integration tests,